docs/source/usingmame/commonissues.rst

```diff
@@ -276,3 +277,14 @@ Note that if you set the autofire button to an input button that's also defined
  If you set button 3 on your controller to autofire and set button 3 to be powerup as well, you will trigger the powerup action every time you grab a powerup because the powerup button is also being held down along with the autofire button.
 
  It is suggested you choose a button for autofire that is not in use for anything else in the current system.
+
+
+.. _gsync-freesync:
+
+Does MAME support G-Sync or FreeSync? How do I configure MAME to use them?
+--------------------------------------------------------------------------
+
+MAME supports both G-Sync and FreeSync right out of the box for Windows and Linux and requires no special configuration. macOS does not support G-Sync or FreeSync.
+
+Be sure to leave triple buffering turned off, as that is unnecessary with variable refresh rate support. Variable Refresh Rate also requires fullscreen mode.
+
```

Comment:
Does G-Sync/FreeSync work with borderless window mode used by `bgfx` on Windows yet?  If not, it should be noted that it only works with OpenGL and the old Direct3D 9.

Reply:
Only have time to make this one quick comment (got a migraine that's going to need special care)

At least NVIDIA does support borderless G-Sync but you have to turn it on specifically in the driver; it defaults to off as it's not completely stable.

Reply:
As I said in the PR, I also plan to revisit and add quite a bit to this after I get the rest of the text worked out to your satisfaction.